### PR TITLE
RO-1088 - [BE] create and update coupons in admin after change @stripe

### PR DESCRIPTION
**Pull request by poxip**

### CODE REVIEW

File: aa_stripe/models.py [hunk 1]
```diff
@@ -131,7 +131,11 @@ class StripeCoupon(StripeBasicModel):
     def __str__(self):
         return self.coupon_id
 
-    def save(self, *args, **kwargs):
+    def save(self, force_retrieve=False, *args, **kwargs):
+        """Save StripeCoupon
```
Comment (jacoor) [inline aa_stripe/models.py]: new lines, please. If multiline comment it looks much better. Also, one line space between first line and the rest is needed.

File: aa_stripe/models.py [hunk 2]
```diff
@@ -397,6 +405,16 @@ class StripeWebhook(models.Model):
     is_parsed = models.BooleanField(default=False)
     raw_data = JSONField()
 
+    def _parse_coupon_notifications(self, action):
```
Comment (jacoor) [inline aa_stripe/models.py]: notification not notifications. It parses one.

File: aa_stripe/models.py [hunk 3]
```diff
@@ -131,7 +131,11 @@ class StripeCoupon(StripeBasicModel):
     def __str__(self):
         return self.coupon_id
 
-    def save(self, *args, **kwargs):
+    def save(self, force_retrieve=False, *args, **kwargs):
+        """Save StripeCoupon
+        Use the force_retrieve parameter to create a new StripeCoupon object form an existing coupon created at
```
Comment (jacoor) [inline aa_stripe/models.py]: well that's not really true. 
Force retrieve will update existing coupon too, no?

Comment (poxip) [inline aa_stripe/models.py]: Yeah, that's true, good point!

**Pull request merged**